pages/src/App.js

```diff
@@ -24,6 +26,6 @@ const App = wrapComponentWithState(({ state }) => {
       <TodoList />
     </div>
   )
-})
+}
 
-export default App;
+export default withState(injectState(App))
```

Comment:
No need to inject state in `App`, not used.

Reply:
You are right, I need to do more attention to my code... 
in coding All my life I used to make things work and thats all.
I‘m learning a lot with you. Thank you